src/components/project-list/project-table.tsx

```diff
@@ -68,7 +75,7 @@ const ProjectTableRow = ({
   };
 
   return (
-    <Row>
+    <Row data-testid="project-card">
```

Comment:
I thought the `data-test-id` attibute had to be unique in the page?
Table can have multiple rows.

Reply:
@michaelrambeau no, `data-testid` does not need to be unique, `id` does.
We can actually query like this 
```js
const allProjects = screen.findAllByTestId("project-card");
```